docs/data-sources/resources.md

```diff
@@ -24,9 +29,14 @@ data "twingate_resources" "foo" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Optional
```

Comment:
This is different behavior then the service_accounts data source. This one doesn't allow me to use the datasource without any options. 
Example
```
data "twingate_resources" "foo" {
}
```
That gives the error
```
│ Error: failed to read twingate_resources
│
│   with module.twingate.data.twingate_resources.foo,
│   on modules/twingate/data.tf line 22, in data "twingate_resources" "foo":
│   22: data "twingate_resources" "foo" {
│
│ Only one of name, name_regex, name_contains, name_exclude, name_prefix or name_suffix must be set.
```

For service_accounts data source it will return all resources.

Reply:
yes, let's discuss it on the call

Reply:
@twingate-blee it's fixed now